parcels/particleset/baseparticleset.py

```diff
@@ -28,6 +29,7 @@ class BaseParticleSet(NDCluster):
     """Base ParticleSet."""
     _collection = None
     kernel = None
+    interaction_kernel = None
```

Comment:
I put it there because it is used in the execute function of `BaseParticleSet`. If that execute function is moved to ParticleSetSOA, then of course this variable does not need to be define in the base.

Reply:
sounds good, I see that.